grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/ShortestPathVertexProgram.java

```diff
@@ -48,35 +52,36 @@
 
     private static final int MAX_ITERATION = 50;
 
-    public static final String FOUND_IN_ITERATION = "shortestPathVertexProgram.foundInIteration";
-    public static final String MIDDLE = "shortestPathVertexProgram.middle";
+    private static final int FROM_SOURCE = 1;
+    private static final int FROM_DESTINATION = -1;
+    private static final int FROM_MIDDLE = 0;
+
+    private static final int DIRECTION = 0;
+    private static final int ID = 1;
```

Comment:
All these magic numbers and strings below could do with a description. Are they keys into something?